test/unit/components/response-body.jsx

```diff
@@ -39,6 +39,13 @@ describe("<ResponseBody />", function () {
     console.warn(wrapper.debug())
     expect(wrapper.find("highlightCodeComponent[canCopy]").length).toEqual(1)
   })
+  
+  it("should render Download file link for text non-empty response", function () {
```

Comment:
This never worked before; even when https://github.com/swagger-api/swagger-ui/pull/8579 is rolled back, this test would not pass.